src/Bridges/Bridges.jl

```diff
@@ -277,7 +277,9 @@ function runtests(
     bridge_supported = all(values(Variable.bridges(model))) do bridge
         return MOI.supports(model, attr, typeof(bridge))
     end
-    if MOI.supports(model, attr, MOI.VariableIndex) && bridge_supported
+    if MOI.supports(model, attr, MOI.VariableIndex) &&
+       bridge_supported &&
+       MOI.get(model, MOI.NumberOfVariables()) > 0
```

Comment:
What is the error without this? Shouldn't it all work? The `ListOfVariableIndices` will be `MOI.VariableIndex[]`.

Reply:
And what is the point of `runtests` if the problem is empty?

Reply:
Good questions :) See https://github.com/jump-dev/SumOfSquares.jl/pull/296/files#diff-49972eb0065227814878ef54ce01433e49bf09e18e6af0ea63d80d000e50bbb4R25-R38, we want to check whether a polynomial is a sum of squares. The vector of coefficients of the polynomial is a `VectorAffineFunction` with no affine term. Because the membership to this set is not so simple, solving such a problem has a point :)

Reply:
See https://github.com/jump-dev/SumOfSquares.jl/actions/runs/9041630162/job/24847221171?pr=296

Reply:
I added the error in the PR description

Reply:
Okaaaay
```julia
julia> import MathOptInterface as MOI

julia> m = MOI.Utilities.Model{Float64}()
MOIU.Model{Float64}

julia> x = MOI.VariableIndex[]
MathOptInterface.VariableIndex[]

julia> MOI.get(m, MOI.VariablePrimalStart(), x)
Union{}[]
```
I think we could fix this somehow.

Reply:
https://github.com/jump-dev/MathOptInterface.jl/issues/2500